### DISCUSSION TITLE
How to create listing no++ in table resource and relation manager?

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#6851] [General]
![07D7F2CA-A12B-47f0-8C18-36A7834BE891](https://github.com/filamentphp/filament/assets/10229660/dd372313-6139-49a1-9f36-a6d277e19b9e)

I want custom number listing of record for each. Thanks before.

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/tables/columns/text#displaying-the-row-index

**Reply by AFSDARIF** [reply]
![image](https://github.com/filamentphp/filament/assets/10229660/8e0f35b3-ee09-4023-99b5-dfec0964558d)
![image](https://github.com/filamentphp/filament/assets/10229660/f5df2a3d-ce93-46f2-b857-c55f59e24df3)

sorry i get error like this,

**Reply by AFSDARIF** [reply]
Ah solved, must be use stdClass;

**Reply by AFSDARIF** [reply]
Thank you
